tests/grpc/config/consumer/ts/yarn.lock

```diff
@@ -0,0 +1,1455 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@grpc/grpc-js@1.9.6":
```

Comment:
Is it important to pin these deps? Curious.. Should we consider not checking in the lock file?

Reply:
Tests like this should include a lock file to keep things as reproducible as possible.

Reply:
I like the stance but most of our tests don't at the moment. I think we're a bit on the fence on this.